tidy3d/components/structure.py

```diff
@@ -42,6 +44,20 @@ class AbstractStructure(Tidy3dBaseModel):
 
     _name_validator = validate_name_str()
 
+    @pydantic.validator("geometry")
+    def _transformed_slanted_polyslabs_not_allowed(cls, val):
+        """Prevents the creation of slanted polyslabs rotated out of plane."""
+        if (
+            isinstance(val, Transformed)
+            and isinstance(val.geometry, PolySlab)
```

Comment:
what if a geometrygroup or clipper operator is transformed, and it contains a slanted polyslab? is it included here already?

Reply:
Good catch, I guess not. We'll need a recursive check in this case. Let me think it through and change it.